manage.py

```
@@ -4,6 +4,50 @@
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "myproject.settings")
+
+    
+    i = 1
+    i = 1
+    # Example Database Password
+    db_password = "htwhtrhthtrhwr"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "db_password"

(S105)